### PR TITLE
Adding Alias

### PR DESCRIPTION
**Pull request by torbenbru**
Reffering to #38 .

### CODE REVIEW

File: library/db2 [hunk 1]
```diff
@@ -59,6 +59,7 @@ class Database:
         self.instance = self.module.params['instance']
         self.automatic = self.module.params['automatic']
         self.path = self.module.params['path']
+	self.alias = self.module.params['alias']
```
Comment (bernardoVale) [inline library/db2]: Can you please fix the spaces?

File: examples/databases.yml [hunk 2]
```diff
@@ -20,6 +20,7 @@
          - name: otherdb
            instance: db2inst1
            automatic: "no"
+	   alias: other
```
Comment (bernardoVale) [inline examples/databases.yml]: Here too :)

File: library/db2 [hunk 3]
```diff
@@ -71,6 +72,7 @@ def main():
             name=dict(required=True, type='str'),
             automatic=dict(required=False, default=True, type='bool'),
             path=dict(required=False, default=None, type='str'),
+	    alias=dict(required=False, default=None, type='str'),
```
Comment (bernardoVale) [inline library/db2]: and here

**Review by bernardoVale — CHANGES_REQUESTED** [review]

**Comment by torbenbru** [comment]
Sure no Idea how the spaces got there..
Edit: should be fixed now

File: library/db2 [hunk 4]
```diff
@@ -95,6 +97,8 @@ def main():
         options += " automatic storage no"
     if db.path:
         options += " on %s" % db.path
+    if db.alias:
+	options += "alias %s " % db.alias
```
Comment (bernardoVale) [inline library/db2:101]: Indentation error :/

**Comment by torbenbru** [comment]
Sorry Github really seems to be buggy with intends for me.. Can you fix this maybe?

**Comment by bernardoVale** [comment]
This has nothing to do with Github. Please open the file with a text editor and fix the space issues. Python language is oriented by indentation.

**Pull request closed without merging**